x/auth/keeper/keeper.go

```diff
@@ -138,6 +137,9 @@ func NewAccountKeeper(
 		AccountNumber: collections.NewSequence(sb, types.GlobalAccountNumberKey, "account_number"),
 		Accounts:      collections.NewIndexedMap(sb, types.AddressStoreKeyPrefix, "accounts", sdk.AccAddressKey, codec.CollInterfaceValue[sdk.AccountI](cdc), NewAccountIndexes(sb)),
 	}
+	if bech32Prefix == "" {
+		ak.bech32Prefix = "bech32 is not used on this chain"
```

Comment:
maybe this string would be better placed in public [Bech32Prefix](https://github.com/cosmos/cosmos-sdk/blob/3d15f9edf295c90672af7c0a735122ddb27d4655/x/auth/keeper/grpc_query.go#L160) API fn if bech32Prefix is empty.